.dockerignore

```diff
@@ -0,0 +1,11 @@
+.git
+.github
+.idea
+.pytest_cache
+.vagrant
+.vscode
+.env
+.env.template
+.DS_Store
+Vagrantfile
+poetry.lock
```

Comment:
We don't want to ignore this `poetry.lock` file - this specifies precisely which dependencies are app is using so it should help us getting better consistency when we run a `poetry install`